frontend/src/services/fileUploadService.js

```diff
@@ -0,0 +1,12 @@
+import axios from 'axios'
+
+export const fileUpload = (file) => {
+    const formData = new FormData();
+    formData.append('file', file);
+    const config = {
+        headers: {
+            'Content-Type': 'multipart/form-data'
```

Comment:
IIRC, setting the content type is not needed with axios, when the data you post is of type `FormData`, not sure..

Reply:
Gonna try this first thing tomorrow, this is just one of the many fixes I tried to fix an Issue I had with the axios-POST so it could as well be obsolete